setup.py

```diff
@@ -10,7 +10,7 @@
 setup(
     name=pkg_name,
     version=version,
-    author="Samuel Garcia, Cole Hurwitz, Jeremy Magland, Alessio Paolo Buccino, Matthias Hennig",
+    author="Alessio Buccino, Cole Hurwitz, Samuel Garcia, Jeremy Magland, Matthias Hennig",
```

Comment:
This is important.

Reply:
eheheh

Reply:
I'm like 4th in all packages. we need to fix this :)